sections/semantics-scriptings.include

```diff
@@ -492,11 +492,17 @@ o............A....e
        content attribute.
   15. <p>Let <var>cryptographic nonce</var> be the element's <a href="#cryptographicnonce">[[<span>CryptographicNonce</span>]]</a>
        internal slot's value.
-  16. Let <var>parser state</var> be "<code>parser-inserted</code>" if the <{script}> element has been flagged
+  16. If the <{script}> element has an <{script/integrity}> attribute, then let <var>integrity metadata</var> be that attribute's value.
+      Otherwise, let <var>integrity metadata</var> be the empty string.
+  17. Let <var>parser metadata</var> be "<code>parser-inserted</code>" if the <{script}> element has been flagged
        as "[=parser-inserted=]", and "`not parser-inserted`" otherwise.
-  17. Let <var>settings</var> be the element's <a>node document</a>'s {{Window}} object's
+  18. Let options be a <a>script fetch options</a> whose <a>cryptographic nonce metadata</a> is 
```

Comment:
a script fetch options -> a set of script...
or -> a script fetch options object